sdk/keyvault/azure-security-keyvault-secrets/README.md

```diff
@@ -282,15 +272,14 @@ secretAsyncClient.getSecret("secretName").subscribe(secret -> {
 
 ### Delete a Secret Asynchronously
 
-Delete an existing Secret by calling `deleteSecret`.
```

Comment:
[here](https://github.com/Azure/azure-sdk-for-java/pull/6061/files#diff-65ce8a0dfab39275230ae65bdb7a6e87L298) should be `listPropertiesOfSecrets`.
Same change for sync secrets and key client too.

Reply:
updated